examples/BlazorServerApp/Program.cs

```diff
@@ -0,0 +1,51 @@
+using BlazorServerApp.Data;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.FeatureManagement;
+
+namespace BlazorServerApp
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+            builder.Services.AddRazorPages();
+            builder.Services.AddServerSideBlazor();
+            builder.Services.AddSingleton<WeatherForecastService>();
+
+            builder.Services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
+                .AddCookie();
+
+            builder.Services.AddScoped<UserAgentContext>();
+
+            builder.Services.AddScopedFeatureManagement()
```

Comment:
Can you describe what's happening here with some comments. Also, it should help draw some attention to this piece since it's one of the things we are trying to exemplify. The same applies to other sections of code that we'd like to exemplify.

Reply:
Sounds good.